appstudio-utils/util-scripts/fetch-test-data.sh

```diff
@@ -23,8 +23,8 @@ for tr in $( pr-get-tr-names $PR_NAME ); do
   data=$( tr-get-result $tr $TEST_NAME )
   if [[ ! -z "${data}" ]]; then
       result_found=1
-      task_name=$( tr-get-task-name ${tr} )
-      echo "${data}" | jq > $( json-data-file test ${task_name} )
+      input_file=$( json-input-file )
```

Comment:
Are you planning to put attestations and tests all in the one input file? That could work, but shorter patch to success here would probably be, something like this:

    # Line 18 or so
    TEST_RESULTS_FILE=$DATA_DIR/test-results.json

    # Line 27
    json-merge-with-key "$data" "$TEST_RESULTS_FILE" $tr test

Reply:
Putting attestations and tests all in one file could work but it requires some extra work in the json-merge-with-key function. I am planning to do that anyway - will probably push a revision shortlyl

Reply:
> Are you planning to put attestations and tests all in the one input file? That could work, but shorter path to success here would probably be something like this:
> 
> ```
> # Line 18 or so
> TEST_RESULTS_FILE=$DATA_DIR/test-results.json
> 
> # Line 27
> json-merge-with-key "$data" "$TEST_RESULTS_FILE" $tr test
> ```

No. not planning on that. Using the input file here was a mistake.